manim/mobject/text/text_mobject.py

```diff
@@ -147,47 +147,29 @@ class Paragraph(VGroup):
     def __init__(self, *text, line_spacing=-1, alignment=None, **config):
         self.line_spacing = line_spacing
         self.alignment = alignment
+        self.consider_spaces_as_chars = config.get("disable_ligatures", False)
         super().__init__()
 
         lines_str = "\n".join(list(text))
         self.lines_text = Text(lines_str, line_spacing=line_spacing, **config)
         lines_str_list = lines_str.split("\n")
         self.chars = self._gen_chars(lines_str_list)
 
-        chars_lines_text_list = self.get_group_class()()
-        char_index_counter = 0
-        for line_index in range(lines_str_list.__len__()):
-            chars_lines_text_list.add(
-                self.lines_text[
-                    char_index_counter : char_index_counter
-                    + lines_str_list[line_index].__len__()
-                    + 1
-                ],
-            )
-            char_index_counter += lines_str_list[line_index].__len__() + 1
-        self.lines = []
-        self.lines.append([])
-        for line_no in range(chars_lines_text_list.__len__()):
-            self.lines[0].append(chars_lines_text_list[line_no])
-        self.lines_initial_positions = []
-        for line_no in range(self.lines[0].__len__()):
-            self.lines_initial_positions.append(self.lines[0][line_no].get_center())
-        self.lines.append([])
-        self.lines[1].extend(
-            [self.alignment for _ in range(chars_lines_text_list.__len__())],
-        )
+        self.lines = [list(self.chars), [self.alignment] * self.chars.__len__()]
+        self.lines_initial_positions = [line.get_center() for line in self.lines[0]]
         self.add(*self.lines[0])
         self.move_to(np.array([0, 0, 0]))
         if self.alignment:
             self._set_all_lines_alignments(self.alignment)
 
     def _gen_chars(self, lines_str_list):
-        """Function to convert plain string to 2d-VGroup of chars. 2d-VGroup mean "VGroup of VGroup".
+        """Function to convert a list of plain strings to a 2d-VGroup of chars.
+        "2d-VGroup" means "VGroup of VGroups".
```

Comment:
I think the wording here is worth improving.

```suggestion
        """Function to convert a list of plain strings to a VGroup containing VGroups.
```

Reply:
True, I wasn't really convinced by the use of "2d-VGroup" in this docstring. Again, I didn't change it so much when I submitted this PR, but again I agree with you on this one.
Maybe I'd add "of chars" at the end of your suggestion, something like
"Function to convert a list of plain strings to a VGroup containing VGroups of chars."